tensorflow/core/kernels/conv_grad_input_ops.cc

```diff
@@ -892,10 +897,13 @@ class Conv2DSlowBackpropInputOp : public OpKernel {
                   errors::NotFound("No algorithm worked!"));
       if (best_result.is_valid()) {
         algorithm_config.set_algorithm(best_result.algorithm());
+        algorithm_config.set_use_tensor_op_math(best_result.use_tensor_op_math());
```

Comment:
Could you add a TODO to merge AlgorithmConfig into PerfResult? We shouldn't have to copy individual field here.

Reply:
The copy of separate parameters is avoided by merging the bool field into AlgorithmType.